docs/source/getting-started.md

```diff
@@ -64,4 +64,13 @@ jupyter dashboards deactivate
 pip uninstall jupyter_dashboards
 ```
 
-Note that there is no command to remove extension assets from the user profile in this version of Jupyter Notebook.
\ No newline at end of file
+### On Jupyter Notebook < 4.0
```

Comment:
These uninstall steps aren't for Notebook < 4.0. They're for any version of Notebook + versions of this extension < 0.4. (Confusing. I know.)